src/microbe_stage/MicrobeHUD.cs

```diff
@@ -712,6 +716,21 @@ public void ShowExtinctionBox()
         extinctionBox.Show();
     }
 
+    public void ShowPatchExtinctionBox(PatchMap map, Species playerSpecies)
+    {
+        if (patchExtinctionBox != null)
+            return;
+
+        winExtinctBoxHolder.Show();
+
+        patchExtinctionBox = (PatchExtinctionBox)patchExtinctionBoxScene.Instance();
+        patchExtinctionBox.Map = map;
+        patchExtinctionBox.PlayerSpecies = playerSpecies;
+        patchExtinctionBox.GoToNewPatch = GoToNewPatch;
+
+        winExtinctBoxHolder.AddChild(patchExtinctionBox);
```

Comment:
Why isn't the patch extinction box always in the win box? I think it should be there, just invisible when not required like the other screens in that holder.